astropy/io/ascii/cds.py

```diff
@@ -188,7 +188,7 @@ def process_lines(self, lines):
                       if x.startswith(('------', '======='))]
         if not i_sections:
             raise core.InconsistentTableError(f'No {self._subfmt} section delimiter found')
-        return lines[i_sections[-1]+1:]  # noqa
```

Comment:
Are you sure it won't complain about the lack of space after colon?

Reply:
With our current flake8 configuration (from setup.cfg), there are no complaints from that whole file.

Reply:
It would be better to ensure all these lines are `black`-compliant.

Reply:
More lines being `black`-compliant already means the diffs `black` will create will be smaller.